mms/context.py

```diff
@@ -77,3 +80,6 @@ def report_status(self, code, reason_phrase=None):
 
     def add_response_property(self, key, value):
         self._response_header[key] = value
+
+    def get_response_property(self):
```

Comment:
Should this not be ```get_response_headers```  ?

Reply:
The field being returned is called "response_property" and other APIs are named as such. So I wouldn't want to change the convention.

Reply:
Actually addressed this. This makes more sense as well.